contrib/python/README.md

```diff
@@ -7,9 +7,33 @@ See [libpod](https://github.com/projectatomic/libpod)
 
 ###
```

Comment:
I don't know what this was here from before, but I recommend restructuring this README to use single-depth nesting changes (so h1 -> h2, instead of h1 -> h3).  The opening h1 is also a bit long, especially when rendered on a phone.  So maybe something like:

```markdown
# podman

A Pythonic library for working with Podman's [Varlink](FIXME-varlink-URL) interface.

## Status: Active Development
 
See [libpod](https://github.com/projectatomic/libpod).
 
## Releases

To build the podman egg:
...
```